### ISSUE TITLE
How to run the calc_log_hist.m and visualize_hist.m?

### ISSUE THREAD

**Issue by iamsiddhantsahu** [yuanxy92/AutoWhiteBalance#3]
I am trying to understand the "Convolutional Color Constancy" paper. Can you please comment and explain how is the log chroma histogram built and how to actually visualize it? 
In the visualize_hist.m function, what is the input X and rho? 
I only want to visualize the 2D log chroma histogram.

**Comment by iamsiddhantsahu** [comment]
Why is 
`uv_0 = -1.421875;` 
in `calc_log_hist.m` ?

**Comment by iamsiddhantsahu** [comment]
In `calc_log_hist.m` what does `function N = Psplat2(u, v, c, bin_lo, bin_step, n_bins)` function does? 
Also in `visualize_hist.m` what does `if ~exist('rho', 'var')
  rho = 0.5;
end` signify?

**Comment by yuanxy92** [comment]
Hi this project is based on ffcc 
Fast Fourier Color Constancy, uv_0 is the initial value

the matlab code is almost based on https://github.com/google/ffcc
and what i do is implementing it using c++/cuda

**Comment by iamsiddhantsahu** [comment]
Yes I know that. But please tell me why is the initial value of `uv_0 = -1.421875`?

**Comment by yuanxy92** [comment]
This value is copied from the ffcc code.
How to choose the initial value is in the ffcc paper.
